### PR TITLE
[Test] Quantize the inference phase of the test.

### PR DESCRIPTION
**Pull request by nadavrot**

### CODE REVIEW

File: tests/unittests/MLTest.cpp [hunk 1]
```diff
@@ -955,11 +983,16 @@ TEST_P(MLTest, testFindPixelRegression) {
...
 #endif
-
 #ifdef GLOW_WITH_OPENCL
 INSTANTIATE_TEST_CASE_P(OpenCL, MLTest, ::testing::Values(BackendKind::OpenCL));
 #endif
+
+INSTANTIATE_TEST_CASE_P(Interpreter, InterpreterAndCPU,
+                        ::testing::Values(BackendKind::Interpreter));
+#ifdef GLOW_WITH_CPU
+INSTANTIATE_TEST_CASE_P(InterpreterAndCPU, MLTest,
```
Comment (rdzhabarov) [inline tests/unittests/MLTest.cpp]: we already instantiate `MLTest` with CPU, what is this for?

Comment (nadavrot) [reply tests/unittests/MLTest.cpp]: I want to disable OpenCL that does not support quantization.

File: tests/unittests/MLTest.cpp [hunk 2]
```diff
@@ -885,15 +889,17 @@ static void generateRegressionTestData(Tensor &images, Tensor &labels) {
 
 /// This is the "Where's Waldo" test. We place a pixel in a tensor and the
 /// network reports the coordinate of the pixel.
-TEST_P(MLTest, testFindPixelRegression) {
+TEST_P(InterpreterAndCPU, testFindPixelRegression) {
+  ExecutionEngine EE{BackendKind::Interpreter};
```
[the remark this answers is not in the document]
Comment (rdzhabarov) [reply tests/unittests/MLTest.cpp]: The thing is that we can only profile Graph on the interpreter.

[the remark this answers is not in the document]
Comment (jfix71) [reply tests/unittests/MLTest.cpp]: Ah whoops didn't realize what was going on/totally missed the setting of the EE to the GetParam() further down.

File: tests/unittests/MLTest.cpp (hunk 1 above)
Comment (nadavrot) [reply tests/unittests/MLTest.cpp]: Good catch!

**Pull request merged**